src/DynamoCore/Extensions/ExtensionLoader.cs

```diff
@@ -124,5 +132,46 @@ private void Log(string msg)
         /// An event that is raised when an extension starts loading.
         /// </summary>
         public event Action<IExtension> ExtensionLoading;
+
+        internal List<string> directoriesToVerifyCertificates = new List<string>();
+
+        private static bool CheckExtensionCertificates(ExtensionDefinition viewExtension)
```

Comment:
Would you add comments for this function for future reference? e.g. why we need this, what case we prevent.